src/renderer/components/ManagerConnect/index.js

```diff
@@ -116,6 +115,7 @@ const Wrapper = styled.div`
   flex: 1;
   align-items: center;
   justify-content: center;
+  padding-bottom: ${p => p.theme.space[6] + p.theme.sizes.topBarHeight};
```

Comment:
beware we use this component in onboarding, in genuine check modal. we don't want this padding here i feel.